_includes/known-issues.html

```diff
@@ -1,17 +1,15 @@
-{% assign page_name = page.title | downcase %}
-{% assign page_slug = page_name | slugify %}
-{% assign github_label = page.github_label_url | default: page.title | replace: ' ', '+' %}
```

Comment:
> [!Note]
> `github_label_url` is not currently used. We might need to bring this back because the github label names do not always line up with the component names, but I would prefer updating the label name instead.